operators/test/e2e/test/elasticsearch/steps_init.go

```diff
@@ -67,7 +67,7 @@ func (b Builder) InitTestSteps(k *test.K8sClient) test.StepList {
 						return err
 					}
 					if err == nil {
-						return fmt.Errorf("elasticsearch %s is still there", b.Elasticsearch.Name)
+						return fmt.Errorf("elasticsearch %s is still there", k8s.ExtractNamespacedName(&stack.Elasticsearch))
```

Comment:
PR #1170 did a massive refactoring of E2E tests, in particular to remove the `stack` concept. So the `stack Stack` became a `b Builder`.